fpdf/fpdf.py

```diff
@@ -1038,6 +1047,18 @@ def set_line_width(self, width):
         if self.page > 0:
             self._out(f"{width * self.k:.2f} w")
 
+    def set_page_background(self, background):
+        """
+        Sets a background color or image to be drawn every time add_page() is called.
+        The method can be called before the first page is created and the value is retained from page to page.
+
+        Args:
+            background: either a string representing a file path to an image, an URL to an image,
+                an io.BytesIO, an instance of `PIL.Image.Image` or an RGB tuple representing a color to fill the background with
+        """
+        self.page_background = background
```

Comment:
Two things:
* we should also accept a `drawing.DeviceRGB`
* I think some type checking should be made here, so that in case of an unsupported value an error is raised now and not later on when `add_page()` is called

Something like this:
```python
if not (isinstance(background, str) or isinstance(background, Image) or isinstance(background, io.BytesIO) or isinstance(background, tuple) or isinstance(background, DeviceRGB)):
	raise TypeError(...)
```